.devcontainer/psinode/configs/dev.config

```diff
@@ -0,0 +1,27 @@
+# psinode configuration
+listen              = 8080
+p2p                 = off
+producer            = myproducer
+host                = psibase.127.0.0.1.sslip.io
+service             = admin-sys.psibase.127.0.0.1.sslip.io:$PREFIX/share/psibase/services/admin-sys
+service             = host.docker.internal:
+admin               = static:*
+database-cache-size = 256MiB
+
+[logger.stderr]
+type   = console
+filter = Severity >= info
+format = [{TimeStamp}] [{Severity}]{?: [{RemoteEndpoint}]}: {Message}{?: {BlockId}}{?RequestMethod:: {RequestMethod} {RequestHost}{RequestTarget}{?: {ResponseStatus}{?: {ResponseBytes}}}}{?: {ResponseTime} µs}
+
+# Log all HTTP reqests to a separate file
+[logger.http]
+type         = file
+filter       = ResponseStatus
+format       = [{TimeStamp}] [{RemoteEndpoint}]: {RequestHost}: {RequestMethod} {RequestTarget}{?: {ResponseStatus}{?: {ResponseBytes}}}
+filename     = http.log
+target       = http-%3N.log
+rotationSize = 64 MiB
+rotationTime = R/2022-10-01T00:00:00Z/P1D
+maxFiles     = 10
+maxSize      = 4 GiB
+flush        = off

```

Comment:
Didn't we conclude that `flush = on` is a better default?

Reply:
fixed